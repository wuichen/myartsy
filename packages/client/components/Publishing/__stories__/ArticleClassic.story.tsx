import { storiesOf } from "@storybook/react"
import React from "react"
import { Article } from "../Article"
import { ClassicArticle } from "../Fixtures/Articles"

storiesOf("Publishing/Articles/Classic", module).add("Classic", () => {
  return <Article article={ClassicArticle} />
})
