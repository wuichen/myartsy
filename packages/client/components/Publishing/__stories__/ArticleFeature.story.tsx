import { storiesOf } from "@storybook/react"
import { SystemContextProvider } from "Artsy"
import { Article } from "Components/Publishing/Article"
import {
  BasicArticle,
  FeatureArticle,
  SeriesArticle,
  SeriesArticleSponsored,
  SuperArticle,
} from "Components/Publishing/Fixtures/Articles"
import {
  HeroSections,
  RelatedCanvas,
} from "Components/Publishing/Fixtures/Components"
import { ArticleData } from "Components/Publishing/Typings"
import { clone, extend } from "lodash"
import React from "react"

storiesOf("Publishing/Articles/Feature/Fullscreen", module)
  .add("Fullscreen", () => {
    return (
      <SystemContextProvider>
        <Article
          article={FeatureArticle}
          relatedArticlesForCanvas={RelatedCanvas}
          showTooltips
        />
      </SystemContextProvider>
    )
  })
  .add("Series", () => {
    const article = clone({
      ...FeatureArticle,
      seriesArticle: SeriesArticle,
      relatedArticles: [BasicArticle, SuperArticle],
    } as ArticleData)

    return (
      <SystemContextProvider>
        <Article article={article} showTooltips />
      </SystemContextProvider>
    )
  })
  .add("Sponsored series", () => {
    const article = clone({
      ...FeatureArticle,
      seriesArticle: SeriesArticleSponsored,
      relatedArticles: [BasicArticle, SuperArticle],
    } as ArticleData)

    return (
      <SystemContextProvider>
        <Article article={article} showTooltips />
      </SystemContextProvider>
    )
  })
  .add("Custom color", () => {
    const article = clone({
      ...FeatureArticle,
      seriesArticle: SeriesArticleSponsored,
      relatedArticles: [BasicArticle, SuperArticle],
    } as ArticleData)

    return (
      <SystemContextProvider>
        <Article
          article={article}
          showTooltips
          backgroundColor="gainsboro"
          color="orangered"
        />
      </SystemContextProvider>
    )
  })

storiesOf("Publishing/Articles/Feature", module)
  .add("Text", () => {
    const article = clone({
      ...FeatureArticle,
      hero_section: {
        type: "text",
        url: FeatureArticle.hero_section.url,
      },
    } as ArticleData)

    return (
      <SystemContextProvider>
        <Article article={article} showTooltips />
      </SystemContextProvider>
    )
  })
  .add("Split", () => {
    const article = clone({
      ...FeatureArticle,
      hero_section: {
        type: "split",
        url: FeatureArticle.hero_section.url,
      },
    } as ArticleData)

    return (
      <SystemContextProvider>
        <Article article={article} showTooltips />
      </SystemContextProvider>
    )
  })
  .add("Basic", () => {
    const article = clone({
      ...BasicArticle,
      sections: [
        {
          type: "text",
          body:
            "<p>The Black Power Tarot was conceived by musician King Khan in consultation with Alejandro Jodorowsky, and designed by illustrator Michael Eaton in 2015. The deck celebrates the strength and achievements of Black musicians, artists, and activists while staying faithful to the imagery and composition of the classic Tarot de Marseilles.</p>",
        },
      ],
    } as ArticleData)

    return (
      <SystemContextProvider>
        <Article
          article={article}
          relatedArticlesForCanvas={RelatedCanvas}
          isTruncated
          showTooltips
        />
      </SystemContextProvider>
    )
  })
  .add("Super Article", () => {
    const article = extend({}, SuperArticle, { hero_section: HeroSections[2] })
    return (
      <SystemContextProvider>
        <Article
          article={article}
          isSuper
          relatedArticlesForCanvas={RelatedCanvas}
          showTooltips
        />
      </SystemContextProvider>
    )
  })
