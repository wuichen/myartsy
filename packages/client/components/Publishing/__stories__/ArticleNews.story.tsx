import { storiesOf } from "@storybook/react"
import { RelatedCanvas } from "Components/Publishing/Fixtures/Components"
import React from "react"
import { Article } from "../Article"
import { NewsArticle } from "../Fixtures/Articles"

storiesOf("Publishing/Articles/News", module)
  .add("Expanded", () => {
    return <Article article={NewsArticle} />
  })
  .add("Collapsed", () => {
    return <Article article={NewsArticle} isTruncated />
  })
  .add("Has Related", () => {
    return (
      <Article
        article={NewsArticle}
        isTruncated
        relatedArticlesForCanvas={RelatedCanvas}
      />
    )
  })
  .add("Mobile Collapsed", () => {
    return <Article article={NewsArticle} isTruncated isMobile />
  })
