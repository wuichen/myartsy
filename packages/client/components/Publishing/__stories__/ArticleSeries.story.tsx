import { storiesOf } from "@storybook/react"
import React from "react"
import { Article } from "../Article"

import {
  SeriesArticle,
  SeriesArticleSponsored,
  StandardArticle,
  VideoArticle,
} from "../Fixtures/Articles"

storiesOf("Publishing/Articles/Series", module)
  .add("Series", () => {
    return (
      <Article
        article={SeriesArticle}
        relatedArticles={[StandardArticle, VideoArticle]}
      />
    )
  })
  .add("Sponsored", () => {
    return (
      <Article
        article={SeriesArticleSponsored}
        relatedArticles={[StandardArticle, VideoArticle]}
      />
    )
  })
