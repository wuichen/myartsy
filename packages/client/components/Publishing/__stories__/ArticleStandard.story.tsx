import { storiesOf } from "@storybook/react"
import { Article } from "Components/Publishing/Article"
import { clone, extend } from "lodash"
import React from "react"

import {
  BasicArticle,
  ImageHeavyStandardArticle,
  MissingVerticalStandardArticle,
  SeriesArticle,
  StandardArticle,
  SuperArticle,
} from "../Fixtures/Articles"

import { SystemContextProvider } from "Artsy"
import {
  HeroSections,
  RelatedCanvas,
  RelatedPanel,
} from "../Fixtures/Components"
import { ArticleData } from "../Typings"

const story = storiesOf("Publishing/Articles/Standard", module)
  .add("Standard", () => {
    return (
      <SystemContextProvider>
        <Article
          article={StandardArticle}
          relatedArticlesForPanel={RelatedPanel}
          relatedArticlesForCanvas={RelatedCanvas}
        />
      </SystemContextProvider>
    )
  })
  .add("Super Article", () => {
    const article = extend({}, SuperArticle, { hero_section: HeroSections[2] })
    return (
      <SystemContextProvider>
        <Article
          article={article}
          relatedArticlesForPanel={RelatedPanel}
          relatedArticlesForCanvas={RelatedCanvas}
          isSuper
        />
      </SystemContextProvider>
    )
  })
  .add("In series", () => {
    const article = clone({
      ...StandardArticle,
      seriesArticle: SeriesArticle,
      relatedArticles: [BasicArticle, SuperArticle],
    } as ArticleData)

    return (
      <SystemContextProvider>
        <Article article={article} />
      </SystemContextProvider>
    )
  })
  .add("Without Vertical", () => {
    return (
      <SystemContextProvider>
        <Article
          article={MissingVerticalStandardArticle}
          relatedArticlesForPanel={RelatedPanel}
          relatedArticlesForCanvas={RelatedCanvas}
        />
      </SystemContextProvider>
    )
  })
  .add("Truncated", () => {
    return (
      <SystemContextProvider>
        <Article
          article={ImageHeavyStandardArticle}
          relatedArticlesForPanel={RelatedPanel}
          relatedArticlesForCanvas={RelatedCanvas}
          isTruncated
        />
      </SystemContextProvider>
    )
  })
  .add("With tooltips", () => {
    return (
      <SystemContextProvider>
        <Article
          article={StandardArticle}
          relatedArticlesForPanel={RelatedPanel}
          relatedArticlesForCanvas={RelatedCanvas}
          showTooltips
        />
      </SystemContextProvider>
    )
  })

story.add(`Multiple articles`, () => {
  const article: ArticleData = {
    ...StandardArticle,
    sections: [
      {
        type: "text",
        body: "<p>What would Antoine Court?</p>",
      },
    ],
  }

  return (
    <SystemContextProvider>
      <div>
        <Article
          article={article}
          relatedArticlesForPanel={RelatedPanel}
          relatedArticlesForCanvas={RelatedCanvas}
          showCollectionsRail
        />
        <Article
          article={article}
          relatedArticlesForPanel={RelatedPanel}
          relatedArticlesForCanvas={RelatedCanvas}
          isTruncated
        />
        <Article
          article={article}
          relatedArticlesForPanel={RelatedPanel}
          relatedArticlesForCanvas={RelatedCanvas}
          isTruncated
        />
      </div>
    </SystemContextProvider>
  )
})
