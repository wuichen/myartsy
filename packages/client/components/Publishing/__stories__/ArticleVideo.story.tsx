import { storiesOf } from "@storybook/react"
import { clone } from "lodash"
import React from "react"
import { Article } from "../Article"
import { ArticleData } from "../Typings"

import {
  SeriesArticle,
  SeriesArticleSponsored,
  StandardArticle,
  VideoArticle,
  VideoArticleSponsored,
} from "../Fixtures/Articles"

storiesOf("Publishing/Articles/Video", module)
  .add("Video", () => {
    return <Article article={VideoArticle} />
  })
  .add("In Series", () => {
    const article = clone({
      ...VideoArticle,
      seriesArticle: SeriesArticle,
    } as ArticleData)

    return (
      <Article
        article={article}
        relatedArticles={[StandardArticle, VideoArticle]}
      />
    )
  })
  .add("With Sponsor", () => {
    return <Article article={VideoArticleSponsored} />
  })
  .add("Series + Sponsored", () => {
    const article = clone({
      ...VideoArticle,
      seriesArticle: SeriesArticleSponsored,
    } as ArticleData)

    return (
      <Article
        article={article}
        relatedArticles={[StandardArticle, VideoArticle]}
      />
    )
  })
