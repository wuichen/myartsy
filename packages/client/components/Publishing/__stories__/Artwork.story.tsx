import { storiesOf } from "@storybook/react"
import React from "react"
import { Artworks } from "../Fixtures/Components"
import { Artwork } from "../Sections/Artwork"

storiesOf("Publishing/Artwork", module)
  .add("Single artist", () => {
    return (
      <div style={{ width: 800 }}>
        <Artwork artwork={Artworks[0]} />
      </div>
    )
  })
  .add("Multiple artists", () => {
    return (
      <div style={{ width: 800 }}>
        <Artwork artwork={Artworks[1]} />
      </div>
    )
  })
  .add("Unlinked", () => {
    return (
      <div style={{ width: 800 }}>
        <Artwork linked={false} artwork={Artworks[1]} />
      </div>
    )
  })
  .add("Missing info", () => {
    return (
      <div style={{ width: 800 }}>
        <Artwork artwork={Artworks[2]} />
      </div>
    )
  })
  .add("Long info", () => {
    return (
      <div style={{ width: 800 }}>
        <Artwork artwork={Artworks[3]} />
      </div>
    )
  })
  .add("Small", () => {
    return (
      <div style={{ width: 400 }}>
        <Artwork artwork={Artworks[1]} />
      </div>
    )
  })
  .add("Fillwidth", () => {
    return (
      <div style={{ width: "100%" }}>
        <Artwork artwork={Artworks[1]} sectionLayout="fillwidth" />
      </div>
    )
  })
  .add("Classic", () => {
    return (
      <div style={{ width: 800 }}>
        <Artwork artwork={Artworks[0]} layout="classic" />
      </div>
    )
  })
