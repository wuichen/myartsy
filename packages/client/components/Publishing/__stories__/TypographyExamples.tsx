import { avantgarde, garamond, unica } from "Assets/Fonts"
import React from "react"
import styled from "styled-components"

export const Typography = () => {
  return (
    <div>
      <GaramondS50>Garamond s50: 50px / 1.1em</GaramondS50>
      <GaramondS40>Garamond s40: 40px / 1.1em</GaramondS40>
      <GaramondS37>Garamond s37: 37px / 1.2em</GaramondS37>
      <GaramondS34>Garamond s34: 34px / 1.5em</GaramondS34>
      <GaramondS30>Garamond s30: 30px / 1.25em</GaramondS30>
      <GaramondS28>Garamond s28: 28px / 1.5em</GaramondS28>
      <GaramondS23>Garamond s23: 23px / 1.5em</GaramondS23>
      <GaramondS19>Garamond s19: 19px / 1.1em</GaramondS19>
      <GaramondS17>Garamond s17: 17px / 1.1em</GaramondS17>
      <GaramondS15>Garamond s15: 15px / 1.25em</GaramondS15>
      <GaramondS11>Garamond s11: 11px / 1.1em</GaramondS11>
      <hr />
      <br />
      <AvantGardeS13>Avant Garde s13: 13px / 1.65em</AvantGardeS13>
      <AvantGardeS11>Avant Garde s11: 11px / 1.65em</AvantGardeS11>
      <hr />
      <UnicaS120>Unica s120: 120px / 1.1em</UnicaS120>
      <UnicaS100>Unica s100: 100px / 1.1em</UnicaS100>
      <UnicaS80>Unica s80: 80px / 1.1em</UnicaS80>
      <UnicaS67>Unica s67: 67px / 1em</UnicaS67>
      <UnicaS65>Unica s65: 65px / 1em</UnicaS65>
      <UnicaS45>Unica s45: 45px / 1.1em</UnicaS45>
      <UnicaS40>Unica s40: 40px / 1.1em</UnicaS40>
      <UnicaS34>Unica s34: 34px / 1.1em</UnicaS34>
      <UnicaS32>Unica s32: 32px / 1.1em</UnicaS32>
      <UnicaS19>Unica s19: 19px / 1.5em</UnicaS19>
      <UnicaS16>Unica s16: 16px / 1.1em</UnicaS16>
      <UnicaS14>Unica s14: 14px / 1.1em</UnicaS14>
      <UnicaS12>Unica s12: 12px / 1.1em</UnicaS12>
    </div>
  )
}

const GaramondS11 = styled.div`
  ${garamond("s11")};
  margin-bottom: 20px;
`
const GaramondS15 = styled.div`
  ${garamond("s15")};
  margin-bottom: 20px;
`
const GaramondS17 = styled.div`
  ${garamond("s17")};
  margin-bottom: 20px;
`
const GaramondS19 = styled.div`
  ${garamond("s19")};
  margin-bottom: 20px;
`
const GaramondS23 = styled.div`
  ${garamond("s23")};
  margin-bottom: 20px;
`
const GaramondS28 = styled.div`
  ${garamond("s28")};
  margin-bottom: 20px;
`
const GaramondS30 = styled.div`
  ${garamond("s30")};
  margin-bottom: 20px;
`
const GaramondS34 = styled.div`
  ${garamond("s34")};
  margin-bottom: 20px;
`
const GaramondS37 = styled.div`
  ${garamond("s37")};
  margin-bottom: 20px;
`
const GaramondS40 = styled.div`
  ${garamond("s40")};
  margin-bottom: 20px;
`
const GaramondS50 = styled.div`
  ${garamond("s50")};
  margin-bottom: 20px;
`

const AvantGardeS11 = styled.div`
  ${avantgarde("s11")};
  margin-bottom: 20px;
`
const AvantGardeS13 = styled.div`
  ${avantgarde("s13")};
  margin-bottom: 20px;
`

const UnicaS12 = styled.div`
  ${unica("s12")};
  margin-bottom: 20px;
`
const UnicaS14 = styled.div`
  ${unica("s14")};
  margin-bottom: 20px;
`
const UnicaS16 = styled.div`
  ${unica("s16")};
  margin-bottom: 20px;
`
const UnicaS19 = styled.div`
  ${unica("s19")};
  margin-bottom: 20px;
`
const UnicaS32 = styled.div`
  ${unica("s32")};
  margin-bottom: 20px;
`
const UnicaS34 = styled.div`
  ${unica("s34")};
  margin-bottom: 20px;
`
const UnicaS40 = styled.div`
  ${unica("s40")};
  margin-bottom: 20px;
`
const UnicaS45 = styled.div`
  ${unica("s45")};
  margin-bottom: 20px;
`
const UnicaS65 = styled.div`
  ${unica("s65")};
  margin-bottom: 20px;
`
const UnicaS67 = styled.div`
  ${unica("s67")};
  margin-bottom: 20px;
`
const UnicaS80 = styled.div`
  ${unica("s80")};
  margin-bottom: 20px;
`
const UnicaS100 = styled.div`
  ${unica("s100")};
  margin-bottom: 20px;
`
const UnicaS120 = styled.div`
  ${unica("s120")};
  margin-bottom: 20px;
`
