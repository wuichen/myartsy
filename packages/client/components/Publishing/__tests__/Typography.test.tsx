import "jest-styled-components"
import React from "react"
import renderer from "react-test-renderer"
import { Typography } from "../__stories__/TypographyExamples"

it("renders properly", () => {
  const typography = renderer.create(<Typography />).toJSON()
  expect(typography).toMatchSnapshot()
})
