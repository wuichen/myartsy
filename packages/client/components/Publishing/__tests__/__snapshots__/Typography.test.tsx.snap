// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders properly 1`] = `
.c10 {
  font-family: 'Adobe Garamond W08','adobe-garamond-pro','AGaramondPro-Regular','Times New Roman',Times,serif;
  font-size: 11px;
  line-height: 1.1em;
  -webkit-font-smoothing: antialiased;
  margin-bottom: 20px;
}

.c9 {
  font-family: 'Adobe Garamond W08','adobe-garamond-pro','AGaramondPro-Regular','Times New Roman',Times,serif;
  font-size: 15px;
  line-height: 1.25em;
  -webkit-font-smoothing: antialiased;
  margin-bottom: 20px;
}

.c8 {
  font-family: 'Adobe Garamond W08','adobe-garamond-pro','AGaramondPro-Regular','Times New Roman',Times,serif;
  font-size: 17px;
  line-height: 1.1em;
  -webkit-font-smoothing: antialiased;
  margin-bottom: 20px;
}

.c7 {
  font-family: 'Adobe Garamond W08','adobe-garamond-pro','AGaramondPro-Regular','Times New Roman',Times,serif;
  font-size: 19px;
  line-height: 1.5em;
  -webkit-font-smoothing: antialiased;
  margin-bottom: 20px;
}

.c6 {
  font-family: 'Adobe Garamond W08','adobe-garamond-pro','AGaramondPro-Regular','Times New Roman',Times,serif;
  font-size: 23px;
  line-height: 1.5em;
  -webkit-font-smoothing: antialiased;
  margin-bottom: 20px;
}

.c5 {
  font-family: 'Adobe Garamond W08','adobe-garamond-pro','AGaramondPro-Regular','Times New Roman',Times,serif;
  font-size: 28px;
  line-height: 1.2em;
  -webkit-font-smoothing: antialiased;
  margin-bottom: 20px;
}

.c4 {
  font-family: 'Adobe Garamond W08','adobe-garamond-pro','AGaramondPro-Regular','Times New Roman',Times,serif;
  font-size: 30px;
  line-height: 1.25em;
  -webkit-font-smoothing: antialiased;
  margin-bottom: 20px;
}

.c3 {
  font-family: 'Adobe Garamond W08','adobe-garamond-pro','AGaramondPro-Regular','Times New Roman',Times,serif;
  font-size: 34px;
  line-height: 1.1em;
  -webkit-font-smoothing: antialiased;
  margin-bottom: 20px;
}

.c2 {
  font-family: 'Adobe Garamond W08','adobe-garamond-pro','AGaramondPro-Regular','Times New Roman',Times,serif;
  font-size: 37px;
  line-height: 1.2em;
  -webkit-font-smoothing: antialiased;
  margin-bottom: 20px;
}

.c1 {
  font-family: 'Adobe Garamond W08','adobe-garamond-pro','AGaramondPro-Regular','Times New Roman',Times,serif;
  font-size: 40px;
  line-height: 1.1em;
  -webkit-font-smoothing: antialiased;
  margin-bottom: 20px;
}

.c0 {
  font-family: 'Adobe Garamond W08','adobe-garamond-pro','AGaramondPro-Regular','Times New Roman',Times,serif;
  font-size: 50px;
  line-height: 1.1em;
  -webkit-font-smoothing: antialiased;
  margin-bottom: 20px;
}

.c12 {
  font-family: 'ITC Avant Garde Gothic W04','AvantGardeGothicITCW01D 731075',AvantGardeGothicITCW01Dm,Helvetica,sans-serif;
  -webkit-font-smoothing: antialiased;
  text-transform: uppercase;
  -webkit-letter-spacing: 1px;
  -moz-letter-spacing: 1px;
  -ms-letter-spacing: 1px;
  letter-spacing: 1px;
  font-size: 11px;
  line-height: 1.65em;
  margin-bottom: 20px;
}

.c11 {
  font-family: 'ITC Avant Garde Gothic W04','AvantGardeGothicITCW01D 731075',AvantGardeGothicITCW01Dm,Helvetica,sans-serif;
  -webkit-font-smoothing: antialiased;
  text-transform: uppercase;
  -webkit-letter-spacing: 1px;
  -moz-letter-spacing: 1px;
  -ms-letter-spacing: 1px;
  letter-spacing: 1px;
  font-size: 13px;
  line-height: 1.65em;
  margin-bottom: 20px;
}

.c25 {
  font-family: Unica77LLWebRegular,Arial,serif;
  -webkit-font-smoothing: antialiased;
  font-size: 12px;
  line-height: 1.4em;
  margin-bottom: 20px;
}

.c24 {
  font-family: Unica77LLWebRegular,Arial,serif;
  -webkit-font-smoothing: antialiased;
  font-size: 14px;
  line-height: 1.4em;
  margin-bottom: 20px;
}

.c23 {
  font-family: Unica77LLWebRegular,Arial,serif;
  -webkit-font-smoothing: antialiased;
  font-size: 16px;
  line-height: 1.1em;
  margin-bottom: 20px;
}

.c22 {
  font-family: Unica77LLWebRegular,Arial,serif;
  -webkit-font-smoothing: antialiased;
  font-size: 19px;
  line-height: 1.5em;
  margin-bottom: 20px;
}

.c21 {
  font-family: Unica77LLWebRegular,Arial,serif;
  -webkit-font-smoothing: antialiased;
  font-size: 32px;
  line-height: 1.1em;
  margin-bottom: 20px;
}

.c20 {
  font-family: Unica77LLWebRegular,Arial,serif;
  -webkit-font-smoothing: antialiased;
  font-size: 34px;
  line-height: 1.1em;
  margin-bottom: 20px;
}

.c19 {
  font-family: Unica77LLWebRegular,Arial,serif;
  -webkit-font-smoothing: antialiased;
  font-size: 40px;
  line-height: 1.1em;
  margin-bottom: 20px;
}

.c18 {
  font-family: Unica77LLWebRegular,Arial,serif;
  -webkit-font-smoothing: antialiased;
  font-size: 45px;
  line-height: 1.2em;
  margin-bottom: 20px;
}

.c17 {
  font-family: Unica77LLWebRegular,Arial,serif;
  -webkit-font-smoothing: antialiased;
  font-size: 65px;
  line-height: 1em;
  margin-bottom: 20px;
}

.c16 {
  font-family: Unica77LLWebRegular,Arial,serif;
  -webkit-font-smoothing: antialiased;
  font-size: 67px;
  line-height: 1em;
  margin-bottom: 20px;
}

.c15 {
  font-family: Unica77LLWebRegular,Arial,serif;
  -webkit-font-smoothing: antialiased;
  font-size: 80px;
  line-height: 1.1em;
  margin-bottom: 20px;
}

.c14 {
  font-family: Unica77LLWebRegular,Arial,serif;
  -webkit-font-smoothing: antialiased;
  font-size: 100px;
  line-height: 1.1em;
  margin-bottom: 20px;
}

.c13 {
  font-family: Unica77LLWebRegular,Arial,serif;
  -webkit-font-smoothing: antialiased;
  font-size: 120px;
  line-height: 1.1em;
  margin-bottom: 20px;
}

<div>
  <div
    className="c0"
  >
    Garamond s50: 50px / 1.1em
  </div>
  <div
    className="c1"
  >
    Garamond s40: 40px / 1.1em
  </div>
  <div
    className="c2"
  >
    Garamond s37: 37px / 1.2em
  </div>
  <div
    className="c3"
  >
    Garamond s34: 34px / 1.5em
  </div>
  <div
    className="c4"
  >
    Garamond s30: 30px / 1.25em
  </div>
  <div
    className="c5"
  >
    Garamond s28: 28px / 1.5em
  </div>
  <div
    className="c6"
  >
    Garamond s23: 23px / 1.5em
  </div>
  <div
    className="c7"
  >
    Garamond s19: 19px / 1.1em
  </div>
  <div
    className="c8"
  >
    Garamond s17: 17px / 1.1em
  </div>
  <div
    className="c9"
  >
    Garamond s15: 15px / 1.25em
  </div>
  <div
    className="c10"
  >
    Garamond s11: 11px / 1.1em
  </div>
  <hr />
  <br />
  <div
    className="c11"
  >
    Avant Garde s13: 13px / 1.65em
  </div>
  <div
    className="c12"
  >
    Avant Garde s11: 11px / 1.65em
  </div>
  <hr />
  <div
    className="c13"
  >
    Unica s120: 120px / 1.1em
  </div>
  <div
    className="c14"
  >
    Unica s100: 100px / 1.1em
  </div>
  <div
    className="c15"
  >
    Unica s80: 80px / 1.1em
  </div>
  <div
    className="c16"
  >
    Unica s67: 67px / 1em
  </div>
  <div
    className="c17"
  >
    Unica s65: 65px / 1em
  </div>
  <div
    className="c18"
  >
    Unica s45: 45px / 1.1em
  </div>
  <div
    className="c19"
  >
    Unica s40: 40px / 1.1em
  </div>
  <div
    className="c20"
  >
    Unica s34: 34px / 1.1em
  </div>
  <div
    className="c21"
  >
    Unica s32: 32px / 1.1em
  </div>
  <div
    className="c22"
  >
    Unica s19: 19px / 1.5em
  </div>
  <div
    className="c23"
  >
    Unica s16: 16px / 1.1em
  </div>
  <div
    className="c24"
  >
    Unica s14: 14px / 1.1em
  </div>
  <div
    className="c25"
  >
    Unica s12: 12px / 1.1em
  </div>
</div>
`;
